go.mod

```diff
@@ -58,3 +58,5 @@ require (
 	golang.org/x/text v0.24.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/onkernel/kernel-go-sdk => github.com/onkernel/kernel-go-sdk v0.11.0
```

Comment:
this should point to 0.11.1 after merging the SDK PR and it should be on line 11 as a normal dependency, not a `replace`